ci/common.rb

```diff
@@ -38,6 +38,26 @@ def install_requirements(req_file, pip_options = nil, output = nil, use_venv = n
   end
 end
 
+def can_skip?
+  return false, [] if Gem.win_platform?
+
+  modified_checks = []
+  `git diff-tree --no-commit-id --name-only -r FETCH_HEAD origin/master`.each_line do |filename|
```

Comment:
Maybe as a future improvment, it could be nice to grab the branch the PR is going to merge into with `TRAVIS_PULL_REQUEST` and the github API.